db/schema.rb

```diff
@@ -362,10 +395,19 @@
     t.datetime "locked_at"
     t.datetime "created_at"
     t.datetime "updated_at"
+    t.boolean  "isActive",                                   default: true
+    t.boolean  "isArchived"
+    t.boolean  "isAdmin",                                    default: false
+    t.boolean  "is_admin",                                   default: false
```

Comment:
You still have the is_admin boolean in the users. This is not needed as you already have the MANAGE_USER policy.